src/app/components/faq/faq.component.html

```diff
@@ -1,6 +1,197 @@
+      <li i18n="faq answer">CLI - a command-line interface to the Engine.</li>
+      <li i18n="faq answer">Bakend - a server that allows you to run Zonemaster tests and save results using a JSON-RPC API and a database.</li>
+      <li i18n="faq answer">GUI - a web interface to the Backend.</li>
+    </ul>
+
+    <p i18n="faq answer">When a domain name (such as 'zonemaster.net') is submitted to Zonemaster (using CLI or GUI), it will verify the domain name’s general health with a series of tests. The tests conducted by Zonemaster can be found in <a href="https://github.com/zonemaster/zonemaster/tree/master/docs/public/specifications/tests#list-of-defined-test-cases">the Defined Test Cases</a> document.</p>
+  </app-faq-question>
+
+  <app-faq-question questionId="who-is-behind-zonemaster">
+    <ng-container title i18n="faq question">
+      Who is behind Zonemaster?
+    </ng-container>
+
+    <p i18n="faq answer">
+      Zonemaster is a joint project between <a i18n-href="Afnic link" href="https://www.afnic.fr/en/">AFNIC</a> (registry of '.fr' TLD and several other TLDs, e.g. '.re', '.pm', '.tf', '.wf', '.yt' and '.paris') and <a i18n-href="IIS link" href="https://internetstiftelsen.se/en/">The Swedish Internet Foundation</a> (registry of '.se' and '.nu' TLDs).
+    </p>
+  </app-faq-question>
+
+  <app-faq-question questionId="how-can-zonemaster-help-me">
+    <ng-container title i18n="faq question">
+      How can Zonemaster help me?
+    </ng-container>
+
+    <p i18n="faq answer">The Zonemaster tool could help different kind of users:</p>
+
+    <ul>
+      <li i18n="faq answer">Users who are knowledgable about the DNS protocol.</li>
+      <li i18n="faq answer">Users who just want to know whether the domain name they own or use have any issues or not.</li>
+    </ul>
+
+    <p i18n="faq answer">Users of the second category should contact their DNS operator in case there are errors or warnings for any test of their domain name.</p>
+  </app-faq-question>
+
+  <app-faq-question questionId="what-makes-zonemaster-different">
+    <ng-container title i18n="faq question">
+      What makes Zonemaster different from other DNS zone validating software?
+    </ng-container>
+
+    <p i18n="faq answer">
+      Firstly, Zonemaster saves all history from earlier tests based on the tested domain name, which means you can go back to a test you did some time ago and compare it to the test you ran just a moment ago.
+    </p>
+    <p i18n="faq answer">
+      Secondly, all tests that Zonemaster runs are defined in Test Case specifications that can be found in the <a href="https://github.com/zonemaster/zonemaster/tree/master/docs/public/specifications/tests#list-of-defined-test-cases">Defined Test Cases</a> document.
+    </p>
+    <p i18n="faq answer">
+      Thirdly, Zonemaster can be used to test undelegated domain names. See <a href="#what-is-an-undelegated-domain-test">Question 12</a>.
```

Comment:
Weird the link is not behaving correctly. I don't know why. I can reproduce on https://zonemaster.fr/fr/faq#what-makes-zonemaster-different . The link leaves the FAQ page which is not the expected behavior.

Reply:
Ah yes, that is because of the base href, it should be fixed now, thanks for noticing.